docs/PersonalApplications.rst

```diff
@@ -314,6 +318,64 @@ The activities in My Activities are similar to those in :ref:`My Activities <Man
 			like on your activity stream page. However, only your connections
 			can see the Activity Composer that enables them to post on your stream.
 
+.. _Managing-Fav-Apps:
+
+=======================================
+Managing your favourite applications
+=======================================
+
+Starting with eXo Platform 6, you can quickly access your favourite applications 
+through the Applications drawer.
+
+To access your favourite applications, you just need to click on this button |AC_image0| from
+the top navigation bar.
+
+|AC_image1|
+
+A drawer will open displaying your favourite applications alphabetically sorted.
+
+|AC_image2|
+ 
+You can reorganize your favorite applications by dragging and dropping their icons according to your choice.
+
+|AC_image3|
+ 
+.. note:: When an administrator sets some applications as default, they also appear in your applications
```

Comment:
```suggestion
.. note:: When an administrator sets some applications as mandatory, they also appear in your applications
```
The "default" option has disappear : https://docs.google.com/document/d/1fRKeiW7kYLMEOdFrL3AWPyJ3k_egp4AwCzdHt8FqRaM/edit#bookmark=id.yj1a1chsqm6y
and is replace by "mandatory"

Reply:
In fact, I planned to put it as "mandatory" when we implement it